packages/frontend/src/components/layout/footer/index.tsx

```diff
@@ -0,0 +1,19 @@
+import './footer.css'
```

Comment:
Yea directory structure doesn't need to be this deep. I'd much rather see `components/footer.tsx` and `components/footer.css`